lib/govuk_publishing_components/presenters/breadcrumbs.rb

```diff
@@ -58,6 +58,13 @@ def tracking_data(breadcrumbs_length)
             dimension28: breadcrumbs_length.to_s,
             dimension29: crumb[:title],
           },
+          ga4_link: {
+            event_name: "navigation",
+            type: "breadcrumbs",
+            text: crumb[:title],
```

Comment:
Does the text property need to be set here or would it be handled automatically by the link tracker?

Reply:
Good spot @JamesCGDS - I've removed that property now :+1: